src/engine/Actor.ts

```diff
@@ -1060,17 +1154,61 @@ export class ActorImpl extends Class implements IActionable, IEvented {
       this.integrate(delta);
 
       // Update actor pipeline (movement, collision detection, event propagation, offscreen culling)
-      for (var trait of this.traits) {
+      for (let trait of this.traits) {
```

Comment:
Should these be `const`?

Reply:
They can be, we should actually add a ts-lint rule eventually to force const where appropriate